lib/collectAndCombine.js

```diff
@@ -166,7 +133,7 @@ function multiplyLikeTerms(node) {
     return status;
   }
 
-  status = Fraction.multiplyFractionsDFS(node);
+  status = Fraction.multiplyFractionsTreeSearch(node);
```

Comment:
Why not just call it multiplyFractions? All the other simplifications that are tree searches are just exported by the module so they are only referred to by their name e.g. when used collectAndCombineLikeTermsTreeSearch is actually just collectAndCombineLikeTerms